libr/anal/fcn.c

```diff
@@ -37,6 +37,16 @@
 
 #define VERBOSE_DELAY if (0)
 
+#define FCN_CONTAINER(x) container_of (x, RAnalFunction, rb)
```

Comment:
Add braces to enforce the execution of this in a single block or make this a function indtead of a macro, maintaibility matters and using that within an if withiut braces will result on unexpected behaviours

Reply:
This is a function-like macro call and has tightest associativity. `(` `)` are unnecessary here